presto-kafka/src/test/java/com/facebook/presto/kafka/TestManySegments.java

```diff
@@ -59,7 +59,7 @@ public void startKafka()
         topicName = "test_" + UUID.randomUUID().toString().replaceAll("-", "_");
 
         Properties topicProperties = new Properties();
-        topicProperties.setProperty("segment.bytes", "256");
+        topicProperties.setProperty("segment.bytes", "1048576");
```

Comment:
Why this change? If it needs to be larger now, I think we also need to change the code below to produce larger messages to ensure many segments. Also, this test seems to be lacking validation that multiple segments are actually created.